helix-rest/src/main/java/org/apache/helix/rest/metadatastore/ZkRoutingDataAccessor.java

```diff
@@ -0,0 +1,67 @@
+package org.apache.helix.rest.metadatastore;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.manager.zk.ZNRecordSerializer;
+import org.apache.helix.manager.zk.client.DedicatedZkClientFactory;
+import org.apache.helix.manager.zk.client.HelixZkClient;
+import org.apache.helix.rest.metadatastore.exceptions.InvalidRoutingDataException;
+
+public class ZkRoutingDataAccessor implements MetadataStoreRoutingDataAccessor {
+  private static final String ROUTING_DATA_PATH = "/METADATA_STORE_ROUTING_DATA";
+  private static final String ZNRECORD_LIST_FIELD_KEY = "ZK_PATH_SHARDING_KEYS";
```

Comment:
Do you see any potentials to use these constants in other classes? If yes, shall we have a separate class as ZKRoutingConstant or something to hold these constants?

Reply:
+1. We could create a constant class.

Reply:
I don't think other classes will use these constants. The first one `ROUTING_DATA_PATH` is used by zookeeper accessor only, and it wouldn't make sense for other classes to use it. The second one is most likely used by this class only, as well.